pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -41,6 +41,28 @@ func checkVersionOfWindowsUpdate() error {
 	return nil
 }
 
+func checkWindowsEdition() error {
+	windowsEditionCmd := `(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID`
+
+	stdOut, _, err := powershell.Execute(windowsEditionCmd)
+	if err != nil {
+		logging.Debug(err.Error())
+		return fmt.Errorf("Failed to get Windows edition")
```

Comment:
`return errors.Wrap(err, "Failed to get Windows edition")`  is better because it doesn't hide the original error.

Reply:
For debugging, the original error is useful, but at the  same time, it goes against showing user-friendly error messages in our UI. So both options are desirable :D We are currently very bad at that anyway, sonot saying one way is better than the other in this case.